From c8b5b91b8a794719280b313e7dd844c3c2142668 Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Fri, 22 Feb 2019 15:10:24 +0200
Subject: [PATCH] statx: don't redefine if structs are available from
 sys/stat.h

---
 src/basic/missing.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/basic/missing.h b/src/basic/missing.h
index dddc6de05c..d6ff612680 100644
--- a/src/basic/missing.h
+++ b/src/basic/missing.h
@@ -1405,7 +1405,7 @@ struct fib_rule_uid_range {
 #define PF_KTHREAD 0x00200000
 #endif
 
-#if ! HAVE_STRUCT_STATX
+#if ! HAVE_STRUCT_STATX && !HAVE_STRUCT_STATX_IN_SYS_STAT_H
 struct statx_timestamp {
         int64_t tv_sec;
         uint32_t tv_nsec;
