From 483c89fa894a477842389da73cec8690669e6dd8 Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Tue, 16 Feb 2021 08:32:03 +0200
Subject: [PATCH] Fix multiple definition of `default_nvbuf_api_version_new'

---
 gstv4l2videodec.c | 2 +-
 gstv4l2videoenc.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/gstv4l2videodec.c b/gstv4l2videodec.c
index 3b69b7a..ef2e3d2 100644
--- a/gstv4l2videodec.c
+++ b/gstv4l2videodec.c
@@ -49,7 +49,7 @@ GST_DEBUG_CATEGORY_STATIC (gst_v4l2_video_dec_debug);
 #define DEFAULT_MAX_PERFORMANCE FALSE
 #define GST_TYPE_V4L2_VID_DEC_SKIP_FRAMES (gst_video_dec_skip_frames ())
 
-gboolean default_nvbuf_api_version_new;
+static gboolean default_nvbuf_api_version_new;
 gint default_num_extra_surfaces;
 
 static gboolean enable_latency_measurement = FALSE;
diff --git a/gstv4l2videoenc.c b/gstv4l2videoenc.c
index 2e26884..414e669 100644
--- a/gstv4l2videoenc.c
+++ b/gstv4l2videoenc.c
@@ -69,7 +69,7 @@ typedef struct
   GstCaps *src_caps;
 } GstV4l2VideoEncCData;
 
-gboolean default_nvbuf_api_version_new;
+static gboolean default_nvbuf_api_version_new;
 #ifdef USE_V4L2_TARGET_NV
 #define DEFAULT_CUDAENC_GPU_ID   0
 #ifdef USE_V4L2_TARGET_NV
