From ae6e0bb4832fe30105eeb539a788f66763f62478 Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Wed, 18 Oct 2017 13:20:25 +0300
Subject: [PATCH] apalis-imx6: lower default resolution on HDMI to 720p

---
 include/configs/apalis_imx6.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/apalis_imx6.h b/include/configs/apalis_imx6.h
index 55bb74ed8a..d90caa43f0 100644
--- a/include/configs/apalis_imx6.h
+++ b/include/configs/apalis_imx6.h
@@ -242,7 +242,7 @@
 		"source ${loadaddr}\0" \
 	"splashpos=m,m\0" \
 	"vidargs=mxc_hdmi.only_cea=1 " \
-		"video=mxcfb0:dev=hdmi,1920x1080M@60,if=RGB24 " \
+		"video=mxcfb0:dev=hdmi,1280x720M@60,if=RGB24 " \
 		"video=mxcfb1:off video=mxcfb2:off video=mxcfb3:off " \
 		"fbmem=32M\0 "
 
