From 86fa457545a2f750a34984106e5863a46dd5ea1e Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Mon, 5 Aug 2019 08:58:14 +0300
Subject: [PATCH] Enable viewer for boot2qt builds

---
 tools/tools.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/tools.pro b/tools/tools.pro
index 7c62eae..a836fe6 100644
--- a/tools/tools.pro
+++ b/tools/tools.pro
@@ -1,7 +1,7 @@
 TEMPLATE = subdirs
 CONFIG += ordered
 
-!boot2qt:!integrity:!qnx {
+!integrity:!qnx {
     SUBDIRS += viewer
 }
 
